roles/openshift_named_certificates/tasks/main.yml

```diff
@@ -29,18 +29,21 @@
   copy:
     src: "{{ item.certfile }}"
     dest: "{{ named_certs_dir }}/{{ item.certfile | basename }}"
+    remote_src: "{{ openshift_named_certificates_from_remote | default(no) }}"
```

Comment:
Set default value in `defaults/main.yml` and use `false` to be consistent with other bools